recipes/mavsdk/all/conanfile.py

```diff
@@ -0,0 +1,167 @@
+from os.path import join
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.files import replace_in_file, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.scm import Git
+from conan.tools.env import Environment
+
+required_conan_version = ">=1.53.0"
+
+
+class MAVSDKConan(ConanFile):
+    name = "mavsdk"
+    license = "BSD-3-Clause"
+    author = "SINTEF Ocean"
+    homepage = "https://mavsdk.mavlink.io/main/en/"
+    url = "https://github.com/mavlink/MAVSDK.git"
+    description = "C++ library to interface with MAVLink systems"
+    settings = "os", "compiler", "build_type", "arch"
+    package_type = "library"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False]
+        }
+    default_options = {
+        "fPIC": True,
+        "shared": False
+        }
+
+    @property
+    def export_sources(self):
+        pass
+        #export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("jsoncpp/[>=1.9.5 <2]")
+        self.requires("tinyxml2/[>=9.0.0 <10]")
+        self.requires("libcurl/[>=7.86.0 <8]")
```

Comment:
For now version ranges are limited to a select set of libraries, check https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges, which means that of these 3, only `libcurl` can use version ranges. Sorry for the confussion :)

Also, if curl's 7.86 is the minimunm required, then great! But the lower the range, the more likely this won't cause issues to other elements on the graph (We have `[>=7.78 <9]` documented as the desired range by default, for example)